test/sql/test_defaults.py

```diff
@@ -1148,6 +1148,12 @@ def go():
             go()
 
     def test_col_w_sequence_non_autoinc_no_firing(self):
```

Comment:
this issue IMO disqualifies this change from being in 1.3.x.   the Sequence() needs to be declared with optional=True if we don't want it to fire off for all sequence-supporting backends.    if someone has a model right now that includes Sequence() on it, so that it works in Oracle, and they rely upon it being ignored in MySQL, that will change here, unless they set optional=True on that Sequence.